Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -123,6 +125,8 @@ def __init__(self, *args, **kwargs):
             self.NON_REDUNDANT_SIZE = 0x400000
         else:
             self.TOP_SWAP_SIZE      = 0x010000
+            if self.NO_OPT_MODE:

```

Comment:
TOP_SWAP_SIZE should be adjusted by build.  It has to match the hardware topswap region size set in FIT.  For QEMU, it is virtual, but should match the QEMU firmware update script.